packages/eslint-mdx/src/parser.ts

```diff
@@ -322,12 +323,20 @@ export class Parser {
 
     const value = node.value as string
 
+    const { loc, start, end } = normalizePosition(node.position)
+
     // fix #4
     if (isComment(value)) {
+      const comment = COMMENT_CONTENT_REGEX.exec(value)[2]
+      this._ast.comments.push({
```

Comment:
Does `_ast` here point to the whole ES AST? If so, it won’t work if a part but not the whole document is ignored maybe?

https://github.com/remarkjs/remark-message-control#markers

Reply:
It is only for `<!-- eslint-disable ruleId -->`.